include/cutlass/device_kernel.h

```diff
@@ -70,13 +74,12 @@ template <typename  T>
 
 /// Generic CUTLASS kernel template.
 template <typename Operator>
-#if defined(CUTLASS_ENABLE_SYCL)
-void Kernel(typename Operator::Params params, char* smem) {
-  // Dynamic shared memory base pointer
-  int* SharedStorageBase = reinterpret_cast<int*>(smem);
-#else
 CUTLASS_GLOBAL
 void Kernel(typename Operator::Params params) {
+#if defined(CUTLASS_ENABLE_SYCL)
```

Comment:
Just out of interest: is there a corresponding change to `syclcompat::launch` for this (to no longer pass the `char*`)?